web/libs/datamanager/src/stores/Tabs/tab.js

```diff
@@ -107,6 +116,9 @@ export const Tab = types
     },
 
     get currentFilters() {
+      if (!isFF(FF_ANNOTATION_RESULTS_FILTERING)) {
```

Comment:
it's not a typo? here you are changing the old behaviour without FF and transforming the old one into new one